### ISSUE TITLE
What to do with CCAnimationCache in v4

### ISSUE THREAD

**Issue by slembcke** [cocos2d/cocos2d-objc#1221]
It's a little awkward to use correctly and should probably be merged into CCAnimation as class methods to make it easier to use. Need to think about this more.

**Comment by andykorth** [comment]
Yeah, I came across CCAnimationCache this weekend. In v4, it seems like it can be entirely replaced with a NSMutableDictionary that holds CCAnimations.
